include/odp/api/spec/schedule.h

```diff
@@ -45,6 +46,95 @@ extern "C" {
  * Maximum schedule group name length in chars including null char
  */
 
+/**
+ * Schedule configuration options
+ */
+typedef struct odp_schedule_config_t {
+
+	/* Number of flows per queue to be supported
+	 * Only valid when scheduler is configured as flow_aware mode
+	 *
+	 * Flows are lightweight entities and packets can be assigned to specific
+	 * flows by the application using odp_event_flow_hash_set() before
+	 * enqueuing the packet into the scheduler.
+	 * This value is valid only when scheduler is configured in flow aware
+	 * mode.
+	 * Depening on the implementation this number might be rounded-off to
+	 * nearest supported value (e.g power of 2)
+	 * This number should be less than maximum flow supported by the
+	 * implementation.
```

Comment:
As we discussed before, if the implementation is going to change the application's input value, then it needs to return to the application the exact number that is being used so the application can properly size its own tables, etc.